src/muya/lib/index.js

```diff
@@ -232,6 +232,10 @@ class Muya {
     this.contentState.listIndentation = listIndentation
   }
 
+  setFrontmatterType (frontmatterType) {
```

Comment:
We set all muya config on `muya.options`, you can modify it by call `muya.setOptions`, so no need to set it on `contentState`?